pkg/registry/apps/statefulset/storage/storage.go

```diff
@@ -92,3 +115,88 @@ var _ rest.ShortNamesProvider = &REST{}
 func (r *REST) ShortNames() []string {
 	return []string{"sts"}
 }
+
+type ScaleREST struct {
+	registry statefulset.Registry
+}
+
+// ScaleREST implements Patcher
+var _ = rest.Patcher(&ScaleREST{})
+
+// New creates a new Scale object
+func (r *ScaleREST) New() runtime.Object {
+	return &extensions.Scale{}
```

Comment:
Is the dummy object in `extensions` necessary for the actual REST API to work? Or are you just saying it's necessary for the current Scale client to work with it?

In other words, if we omit the dummy object, and do a direct `curl` to `/apis/apps/v1beta2/namespaces/default/statefulsets/blah/scale`, would that work?